alter_configs_response.go

```diff
@@ -0,0 +1,95 @@
+package sarama
+
+import "time"
+
+type AlterConfigsResponse struct {
+	ThrottleTime time.Duration
+	Resources    []*AlterConfigsResourceResponse
+}
+
+type AlterConfigsResourceResponse struct {
+	ErrorCode int16
+	ErrorMsg  string
```

Comment:
🤔 Maybe I should squash `ErrorCode` and `ErrorMsg` into a `KError`

Reply:
It’s not clear to me from the docs whether this *is* a KError or if it’s  some other error status set? Normally KErrors don’t include a string in the actual message, just the code.